src/Loader/web/Loader.js

```diff
@@ -0,0 +1,19 @@
+import styled, { keyframes } from 'styled-components';
+
+const spin = keyframes`
+  from {transform: rotate(0deg);}
+  to {transform: rotate(360deg);}
+`;
+
+const Loader = styled.div`
+  display: inline-block;
+  animation: ${spin} 1s linear infinite;
+  width: ${(props) => props.theme.px(5)};
+  height: ${(props) => props.theme.px(5)};
+  border-radius: 50%;
+  border: 4px solid ${(props) => props.theme.color.white};
```

Comment:
`border` and `border-color` can also vary based on different applications implementation